Presentation.Web/Tests/Helpers/SelectHelper.ts

```diff
@@ -0,0 +1,11 @@
+﻿
+class SelectHelper {
+
+    public static openAndSelect(selectId: string, optionLabel: string) {
+        console.log("Opening select and selecting");
```

Comment:
du kunne godt skrive dine parametre i log beskeden for så forstår man bedre konteksten